bazel/cc_grpc_library.bzl

```diff
@@ -0,0 +1,61 @@
+"""Generates and compiles C++ grpc stubs from proto_library rules."""
```

Comment:
What are the difference between bzl extension in this PR vs the one already in grpc github repo (https://github.com/grpc/grpc/blob/master/grpc.bzl)? And why these grpc bzl files part of googleapis not grpc?